Cargo.toml

```diff
@@ -29,7 +29,9 @@ config = "0.13.3"
 env_logger = "0.10.0"
 hex = "0.4.3"
 jwt-simple = "0.11"
+kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev="0d8146321c3de023f0f7f40e47fc0f860133dfc7", default-features = false }
 kbs-types = "0.6.0"
+kms = { git = "https://github.com/confidential-containers/guest-components.git", rev="0d8146321c3de023f0f7f40e47fc0f860133dfc7", default-features = false }
```

Comment:
This is a little awkward. As `kbs_protocol` and `kms` are both from `guest-components`, they might share some deps with same version. Different versions of the both might potentially cause the crate dependency version conflict when doing `cargo build/check`.

Thus, for ease, I move the two crates into the project's `Cargo.toml` and line-by-line. This would help when we want to update one of them -- we do not need to search into the code where the other is.